Ingest Support charter

lauraclarke opened the pull request:
~~**October 18**: Last call for community reviews~~
~~**October 25**: Last call for oversight reviews~~
**November 15**: Approved during DCP PM call

lauraclarke committed:
This is a place holder for the ingest support charter

lauraclarke committed:
First draft of ingest support charter

brianraymor committed:
Minor editorial updates

Removed template _artifacts_

brianraymor committed:
Minor editorial changes

brianraymor commented on charters/IngestSupport/charter.md:
```diff
@@ -0,0 +1,100 @@
+
+## Description
+
+The Ingest Support team acts as a liaison between labs who want to contribute data to the Human Cell Atlas (HCA) and the Data Coordination Platform (DCP). This team works with data contributing labs, to understand each lab's data and ensure the experiments are well-represented in the platform and to support the lab writing data to the platform, working with them to resolve any issues in the proce ...
```
What does *experiments are well-represented* mean in this context? Good? Sufficient? _accurately represented_ is used below in **Objectives** which is clearer.

Editorial thoughts:

_The Ingest Support team is the liaison between the Data Coordination Platform and labs contributing data to the Human Cell Atlas (HCA). This team closely collaborates with each data contributor to:_

•	_Develop a deep understanding of their lab data_
•	_Ensure that their experiments are well-represented in the DCP_
•	_Help submit their lab data to the DCP, resolving any emerging issues during the process_

lauraclarke replied:
Thanks, I have taken your proposal and made some updates. I have but a caveated the statement as in a years time we are unlikely to spend a lot of time with labs who are generating well-understood data types like 10x or SmartSeq 2 on regularly submitted sample types (say the immune system) because hopefully, we won't have the capacity

brianraymor commented on charters/IngestSupport/charter.md:
```diff
@@ -0,0 +1,100 @@
+# Ingest Support
```
_Ingest Assist[ance] (?)_ to avoid _Ingest Support **supports**_ ... or _Ingest Central_ which I've loved every since I discovered the repo.

lauraclarke replied:
I am going to change the usage of support as a verb to assist as I prefer Support as the group name it being an actual noun

brianraymor commented on charters/IngestSupport/charter.md:
```diff
@@ -0,0 +1,100 @@
+
+*Data contributor* - any user who uploads data and metadata into the DCP
+
+*Data wrangler* - a member of the ingest support team who supports to data contributors wanting to upload data into the DCP
```
Bit redundant. How about:

_- the Ingest Support team member who assists labs contributing data to the HCA._

brianraymor commented on charters/IngestSupport/charter.md:
```diff
@@ -0,0 +1,100 @@
+
+*Submission* - the grouping of data and metadata that makes sense to a data contributor. For example, a data contributor might typically upload all the data at once for a publication they have in preparation. This grouping does not necessarily correspond to how data is stored or displayed in the DCP and how it should be displayed to a data consumer
+
+*Data* - raw data files from an experiment such as fastq files for sequencing experiments or tiff stacks for imaging experiments
```
A link to references would not be remiss? - for example `[fastq](https://en.wikipedia.org/wiki/FASTQ_format)` or `[fastq](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2847217/)`

lauraclarke replied:
Added for sequencing, @hewgreen @zperova could you suggest a suitable link for the imaging example?

brianraymor commented on charters/IngestSupport/charter.md:
```diff
@@ -0,0 +1,100 @@
+
+*Data wrangler* - a member of the ingest support team who supports to data contributors wanting to upload data into the DCP
+
+*Submission* - the grouping of data and metadata that makes sense to a data contributor. For example, a data contributor might typically upload all the data at once for a publication they have in preparation. This grouping does not necessarily correspond to how data is stored or displayed in the DCP and how it should be displayed to a data consumer
```
I'd move *Submission* after *Data* and *Metadata* since it relies on those definitions.

brianraymor commented on charters/IngestSupport/charter.md:
```diff
@@ -0,0 +1,100 @@
+
+*Metadata* - descriptive information about what projects, samples and experiments are and how they were collected or conducted
+
+*New data types* - experimental data types which either can't already be ingested by the DCP or have only recently (~6m - 1yr) started to be ingested by the system
```
What is an _**experimental** data type_? Is it a data type used in an experiment -or- is the data type itself _experimental_? I became confused when I read about established experimental data types below.

lauraclarke replied:
That is reasonable. I mean a data type generated by an experiment. I will see if I can improve the phrasing to make this clearer

lauraclarke replied:
For now I have left the phrasing the same but changed the word to assay

An alternative would be to have

New data types - data types generated by experimental assays that either can't already be ingested by the DCP or have only recently (~6m - 1yr) started to be ingested by the system

brianraymor commented on charters/IngestSupport/charter.md:
```diff
@@ -0,0 +1,100 @@
+- Updating the HCA metadata schemas 
+- Answering HCA helpdesk questions that come to data-help@humancellatlas.org about non-data contribution topics
+
+## Milestones and Deliverables
```
_Coming soon?_
_Intentionally blank?_

lauraclarke replied:
Coming soon

brianraymor commented on charters/IngestSupport/charter.md:
```diff
@@ -0,0 +1,100 @@
+## Objectives
+
+- Supporting data contributors uploading data and metadata to the DCP
+- Helping data contributors fix errors with their data and metadata before it can be submitted to the DCP
```
What kinds of *errors* and perhaps _address errors_

lauraclarke replied:
I have switched to address as that is indeed a better term

I have switched data to data format as these are syntax/structure errors rather than data quality issues. 

I have left metadata as metadata as there is already format checks and on the roadmap is and both graph and semantic validation so that is a bit more blurry on structure vs quality

brianraymor commented on charters/IngestSupport/charter.md:
```diff
@@ -0,0 +1,100 @@
+- Supporting data contributors uploading data and metadata to the DCP
+- Helping data contributors fix errors with their data and metadata before it can be submitted to the DCP
+- Ensuring a data contributor's experiment can be accurately represented in the DCP
+- Working with the data contributor to propose extensions to the HCA metadata schema if additional metadata fields are required
```
Regarding _Working with_ - Does the contributor help with the proposal -or- is Ingest Support proposing extensions to address the needs of the contributor based on your deep understanding of their data?

Such as:

_Proposing enhancements to the HCA DCP metadata schema if additional metadata fields are required by the data contributor_

lauraclarke replied:
This will be a variable process depending on the newness of the data, the level of understanding of the data contributor and how engaged the contributor is in the process.

The SpaceTX folks are actively engaged  and working with Zina and Matt to define and review the spec though I don't know if they have actually made direct metadata requests in the git sense

Many of our data contributors don't know what they could be giving us and in those cases, it is very much wrangler driven.

I don't want to write this in a way which precludes either scenario. Would two statements to reflect this situation be better.

I have made an update, I am not sure if it is better or not

brianraymor commented on charters/IngestSupport/charter.md:
```diff
@@ -0,0 +1,100 @@
+### Documentation and Training
+
+- Writing training material about how to submit data to the DCP
+- Responding to data contribution requests on Zendesk
```
Why is _Responding to data ... requests_ under **Documentation and Training**?

lauraclarke replied:
I was trying to avoid duplication but then I broke that rule later. I have now merged this with another group of duplicates under all data contributors

brianraymor commented on charters/IngestSupport/charter.md:
```diff
@@ -0,0 +1,100 @@
+### User Experience
+
+- Collecting feedback about the submission process from data contributors
+- Working with the UX team to use feedback to define new requirements for the system to be past to the PM team
```
_to be **past** to_ -> _to be passed to_ or _to be prioritized by_

malloryfreeberg replied:
Also spell out PM if it's not defined earlier.

lauraclarke replied:
Updated

brianraymor commented on charters/IngestSupport/charter.md:
```diff
@@ -0,0 +1,100 @@
+
+### Scientific "guardrails" 
+
+The ingest support team will work with the HCA DCP PM team and the HCA scientific leadership to understand the data type priorities for the HCA and help communicate those to potential data contributors
```
The _**I**ngest **S**upport team will identify data type priorities in collaboration with the DCP PM team and the HCA Science Leadership to communicate to potential data contributors._

lauraclarke replied:
Updated

lauraclarke committed:
Changes to description and switching support to assist

This is mostly an update to the description both to clarify the meaning and improve readability. Also switched support to assist in the context of it being used as a verb to avoid the clunky ingest support supports

malloryfreeberg commented on charters/IngestSupport/charter.md:
```diff
@@ -0,0 +1,104 @@
+
+The Ingest Support team is the liaison between the Data Coordination Platform (DCP) and labs contributing data to the Human Cell Atlas (HCA). This team closely collaborates with each data contributor to:
+
+*	Working with data contributors who are generating new data types to develop a deep understanding of their data
```
In first point, replace tab with space after the asterisk to be consistent with second two points.

lauraclarke replied:
I don't see the tab. I have deleted and readded spaces. If this doesn't fix it you will need to show me what you see

malloryfreeberg commented on charters/IngestSupport/charter.md:
```diff
@@ -0,0 +1,104 @@
+
+*	Working with data contributors who are generating new data types to develop a deep understanding of their data
+* Assisting data contributors to describe their experiments in a manner that supports finability, reusability and reproducability with in the DCP ecosystem
+* Helping data contributors to submit their data to the DCP, resolving any emerging issues during the process
```
The -ing should be removed from the first word of each bullet point so the combined sentences read:
"...collaborates with each data contributor to _work_ with..."
"...collaborates with each data contributor to _assist_ data..."
"...collaborates with each data contributor to _help_ data..."

lauraclarke replied:
Thes statements are intentionally using "ing" to be in the active voice

malloryfreeberg commented on charters/IngestSupport/charter.md:
```diff
@@ -0,0 +1,104 @@
+The Ingest Support team is the liaison between the Data Coordination Platform (DCP) and labs contributing data to the Human Cell Atlas (HCA). This team closely collaborates with each data contributor to:
+
+*	Working with data contributors who are generating new data types to develop a deep understanding of their data
+* Assisting data contributors to describe their experiments in a manner that supports finability, reusability and reproducability with in the DCP ecosystem
```
s/finability/fin**d**ability/
s/with in/within/

lauraclarke replied:
fixed

malloryfreeberg commented on charters/IngestSupport/charter.md:
```diff
@@ -0,0 +1,104 @@
+
+*Data contributor* - any user who uploads data and metadata into the DCP
+
+*Data wrangler* - a member of the ingest support team who assists to data contributors wanting to upload data into the DCP
```
s/ingest support team/**I**ngest **S**upport team/
s/assists to/assists/

lauraclarke replied:
fixed

malloryfreeberg commented on charters/IngestSupport/charter.md:
```diff
@@ -0,0 +1,104 @@
+
+*Data wrangler* - a member of the ingest support team who assists to data contributors wanting to upload data into the DCP
+
+*Submission* - the grouping of data and metadata that makes sense to a data contributor. For example, a data contributor might typically upload all the data at once for a publication they have in preparation. This grouping does not necessarily correspond to how data is stored or displayed in the DCP and how it should be displayed to a data consumer
```
I would remove _typically_ as I don't think we want to claim to know how users likely will act. Better to just illustrate with an example of what someone _might_ do.

lauraclarke replied:
rephrased

malloryfreeberg commented on charters/IngestSupport/charter.md:
```diff
@@ -0,0 +1,104 @@
+
+*New data types* - experimental data types which either can't already be ingested by the DCP or have only recently (~6m - 1yr) started to be ingested by the system
+
+*Established data types* - experimental data types which have a history of submission and downstream use in the DCP and there are already many datasets of this type represented in the platform
```
s/DCP and there are/DCP and _for which_ there are/

lauraclarke replied:
fixed

malloryfreeberg commented on charters/IngestSupport/charter.md:
```diff
@@ -0,0 +1,104 @@
+## Objectives
+
+- Assisting data contributors uploading data and metadata to the DCP
+- Helping data contributors fix errors with their data and metadata before it can be submitted to the DCP
```
I think this was mentioned somewhere else in the document, but it might sound better to say "Helping data contributors _address_ errors with their data and metadata" instead of "Helping data contributors _fix_ errors with their data and metadata"

lauraclarke replied:
I think I made this edit consistently based on Brian's recommendation but please do check

malloryfreeberg commented on charters/IngestSupport/charter.md:
```diff
@@ -0,0 +1,104 @@
+### Documentation and Training
+
+- Writing training material about how to submit data to the DCP
+- Responding to data contribution requests on Zendesk
```
- Writing/delivering training material about how to submit data to the DCP
- Writing/maintaining documentation about how to submit data to the DCP

lauraclarke replied:
Changed, there has also been some rearrangement here

lauraclarke committed:
Updated based on Brian Raymor's comments

lauraclarke committed:
Updated based on Mallory's review

lauraclarke committed:
Update slack written urls to be consistently formatted

brianraymor commented on charters/IngestSupport/charter.md:
```diff
@@ -0,0 +1,107 @@
+## Description
+
+The Ingest Support team is the liaison between the Data Coordination Platform (DCP) and labs contributing data to the Human Cell Atlas (HCA). This team closely collaborates with each data contributor to:
+
```
_to Working_
_to Assisting_
_to Helping_

doesn't scan. Maybe simplify the second sentence to:

 _This team is:_
-_Working_
-_Assisting_

brianraymor commented on charters/IngestSupport/charter.md:
```diff
@@ -0,0 +1,107 @@
+
+*Submission* - the grouping of data and metadata that makes sense to a data contributor to submit at once time. For example, a data contributor might upload all the data at once for a publication they have in preparation. This grouping does not necessarily correspond to how data is stored or displayed in the DCP and how it should be displayed to a data consumer
+
+*The PM team - the team responsible for the oversight of the DCP as the product, developing the shared understanding of the overall product vision, priorities, and release schedule with the community.
```
Need to close the italic markdown. `*The PM team*`

brianraymor commented on charters/IngestSupport/charter.md:
```diff
@@ -0,0 +1,107 @@
+
+*Established data types* - assay data types which have a history of submission and downstream use in the DCP and for which there are already many datasets of this type represented in the platform
+
+*Submission* - the grouping of data and metadata that makes sense to a data contributor to submit at once time. For example, a data contributor might upload all the data at once for a publication they have in preparation. This grouping does not necessarily correspond to how data is stored or displayed in the DCP and how it should be displayed to a data consumer
```
submit at *one* time

brianraymor commented on charters/IngestSupport/charter.md:
```diff
@@ -0,0 +1,107 @@
+- Helping data contributors address errors with their data formatting and metadata before it can be submitted to the DCP. 
+- Ensuring a data contributor's experiment can be accurately represented in the DCP
+- Working with the data contributor to understand if new metadata fields are needed to represent their data
+- Proposing enhancements to the metadata schemas following the process defined by the metadata team
```
In general, I would probably capitalize the first letter of team names like Ingest Infrastructure. Metadata is also Metadata Schema team based on their charter?

lauraclarke replied:
I think I got all of them

lauraclarke committed:
updates based on Brian's suggestions

lauraclarke committed:
changing metadata team to metadata schema team

brianraymor commented on charters/IngestSupport/charter.md:
```diff
@@ -0,0 +1,106 @@
+- Ensuring a data contributor's experiment can be accurately represented in the DCP
+- Working with the data contributor to understand if new metadata fields are needed to represent their data
+- Proposing enhancements to the metadata schemas following the process defined by the Metadata Schema team
+- Working with the UX team to collect and provide feedback from data contributors to the Ingest Infrastructure team about how the infrastructure can be improved
```
Although Tony describes his charter as _This PR is for the first draft of the ingest **infrastructure** charter_, the actual name he selected was Ingestion Service.

It will probably be helpful to compare Ingestion Service with Ingest Support to ensure that there's a common style between charters

brianraymor commented on charters/IngestSupport/charter.md:
```diff
@@ -0,0 +1,106 @@
+
+*Data contributor* - any user who uploads data and metadata into the DCP
+
+*Data wrangler* -  the Ingest Support team member who assists labs contributing data to the HCA.
```
In Ingest Service, @tburdett defines:

_Data wrangler - a data contributor who works as part of the HCA consortium to provide assistance to biologists wanting to upload data into the DCP_

Can we align one definition across charters?

brianraymor commented on charters/IngestSupport/charter.md:
```diff
@@ -0,0 +1,106 @@
+- Assisting data contributors uploading data files to the platform
+- Assisting data contributors using the ingest user interface to generate a metadata template spreadsheet
+- Assisting data contributors uploading metadata to the platform
+- Assisting data contributors who wish to use the ingest API to submit data
```
Ingest API to match the style in Ingestion Service ...

I'm wondering whether we want to simply use DCP in place of _platform_

morrisonnorman committed:
Additions to Milestones and Deliverables

morrisonnorman committed:
fix scNuc-Seq typo

kbergin commented:
Hello!

Would you like to add a reference to the new upload team email on this charter?

On the Data Processing Charter I decided to make the title of the charter `mailto:email@data.humancellatlas.org` and to add it to the Communication section. I formatted it as
`[Team email](mailto:email@data.humancellatlas.org): email@data.humancellatlas.org`

For your reference, your team email is `wrangler-team@data.humancellatlas.org`

Thanks! I'll be posting this on each charter for convenience, please do let me know your thoughts :)

claymfischer committed:
Update charter.md

Added contact in same style as @kbergin.

lauraclarke committed:
Small formatting change to communications section

malloryfreeberg committed:
Minor grammar edits

malloryfreeberg committed:
Minor grammar edits.

malloryfreeberg committed:
Replace platform with DCP

brianraymor committed:
Editorial updates to Communication section

brianraymor committed:
Editorial updates to Github repositories section
